Modelica/package.mo

```diff
@@ -1300,6 +1300,49 @@ be defined like:
       tab=\"Controller\",
       group=\"Speed control\"));
 </pre></blockquote>
+
+<h4>Whitespace and Indentation</h4>
+<p>
+Trailing white-space (i.e., white-space at the end of the lines) shall not be used.
+The tab-character shall not be used, since the tab-stops are not standardized.
+</p>
+<p>
+The code in a&nbsp;class shall be indented relative to the class in steps of two spaces;
```

Comment:
Have we adopted some convention to always follow _a_ by a non-breaking space?  If not, I suggest using a normal space here:
```suggestion
The code in a class shall be indented relative to the class in steps of two spaces;
```

Reply:
I guess this is based on my changes to the documentation in the last years - I use it whenever improving the documentation. 
IMO we have no such rule but this is a common typography rule.

Reply:
Seems so, it was also used in #4216 explaining tabs and groups (that you reviewed).

Reply:
Sure it may have slipped by in some other review.  Resolving, as it turned out to intentional and not introduced for the first time in this PR.